packages/terra-data-grid/tests/jest/ColumnHeader.test.jsx

```diff
@@ -84,10 +83,6 @@ describe('ColumnHeader', () => {
     expect(columnHeader).toHaveLength(1);
     expect(columnHeader.props().height).toBe('3rem');
 
-    // Validate ColumnHeaderCell React component
```

Comment:
Why was this removed? Shouldn't this exist?

Reply:
Good catch, added these back here: 9cb7c6db2d658fe3b335b1ed87c0c867b887ad13